### DISCUSSION TITLE
MONOREPO: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"

### DISCUSSION THREAD

**Discussion by glitch-txs** [vercel/community#5681] [Bug Report]
### Page to Investigate

https://github.com/roninbuilders/connectors

### Steps to Reproduce

Hi, I have a very simple monorepo which consists of:
- One package
- One Next.js 14 project inside a playground folder

I'm just using pnpm workspaces and typescript for transpiling the package.

when I run locally:

- pnpm install
- pnpm run build
- pnpm run dev:playground

everything works fine.

but I have an issue in Vercel when I try to deploy the playground. After the installation it throws the following error:

```sh
Warning: Could not identify Next.js version, ensure it is defined as a project dependency.
--
17:35:21.915 | Error: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"
```

These are my settings:
![image](https://github.com/vercel/community/assets/66949816/1b818215-53e5-4cc8-9f74-d17e854a855e)
![image](https://github.com/vercel/community/assets/66949816/c8513760-2a09-45f7-a1e0-104629ba6c6b)

I have also tried to use the actual path of the playground but this prevents from the monorepo to build the package so it fails during build as it cannot find the package module.

**Comment by jeremiahhaastrup** [comment]
were you able to find a solution?

**Reply by glitch-txs** [reply]
I ended up installing next as devDep I think :/
